Client/game_sa/CModelInfoSA.h

```diff
@@ -291,6 +302,7 @@ class CModelInfoSA : public CModelInfo
     RpClump*                                                                     m_pCustomClump;
     static std::map<unsigned short, int>                                         ms_RestreamTxdIDMap;
     static std::map<DWORD, float>                                                ms_ModelDefaultLodDistanceMap;
+    static std::map<DWORD, unsigned short>                                       ms_ModelDefaultFlagsMap;
```

Comment:
Why not combine these two into one?
Also, I feel like an `std::map` has a bigger memory overhead(3 pointers / key value pair), than just a large fixed size array(which also would be a lot faster, and would be less of a hassle).
What do others think?
Perhaps merge it like this, and let me open a separate PR merging them into an `std::array`?